terms.html

```diff
@@ -149,25 +151,33 @@
 
   <dd>
 The formal syntax of a <a>decentralized identifier</a>. The generic DID scheme
-begins with the prefix <code>did:</code> as defined in <a href="#did-syntax"></a>.
-Each <a>DID method</a> specification must define a specific DID
-scheme that works with that specific <a>DID method</a>. In a specific DID method
-scheme, the DID method name must follow the first colon and terminate with the
-second colon, e.g., <code>did:example:</code>
+begins with the prefix <code>did:</code> as defined in <a
+href="#did-syntax"></a>. Each <a>DID method</a> specification defines a specific
+DID scheme that works with that specific <a>DID method</a>. In a specific DID
+method scheme, the DID method name follows the first colon and terminates with
+the second colon, e.g., <code>did:example:</code>
   </dd>
 
   <dt><dfn data-lt="DID subjects">DID subject</dfn></dt>
 
   <dd>
-The entity identified by a <a>DID</a> and described by a <a>DID document</a>. A
-<a>DID</a> has exactly one DID subject. Anything can be a DID subject: person,
-group, organization, physical thing, digital thing, logical thing, etc.
+The entity identified by a <a>DID</a> and described by a <a>DID document</a>.
+Anything can be a DID subject: person, group, organization, physical thing,
+digital thing, logical thing, etc.
+  </dd>
+
+  <dt><dfn data-lt="service">DID services</dfn></dt>
```

Comment:
I'd rather not change this from "service" to "DID services", to be consistent with the section https://w3c.github.io/did-core/#services. Just like we also say "verification methods", not "DID verification methods".